rest/index.py

```diff
@@ -521,12 +521,12 @@ def kibanaDashboards():
     try:
         es_client = ElasticsearchClient((list('{0}'.format(s) for s in options.esservers)))
         search_query = SearchQuery()
-        search_query.add_must(TermMatch('_type', 'dashboard'))
+        search_query.add_must(TermMatch('type', 'dashboard'))
```

Comment:
If you create a dashboard in the new kibana, does it set the type to `dashboard` automatically?

Reply:
I just read the overflow of the description with the title, so I think that's a yes, that's neat!